src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/SelfLogLogger.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Exceptionless.Logging;
+using Serilog.Debugging;
+
+namespace Serilog.Sinks.Exceptionless {
+    public class SelfLogLogger : IExceptionlessLog {
```

Comment:
Will this class name conflict with other self log implementations? Should we prefix it?